stock_orderpoint_generator/models/product.py

```diff
@@ -26,53 +27,23 @@ def create(self, vals):
             record.auto_orderpoint_template_ids.create_orderpoints(record)
         return record
 
-    @api.multi
     def write(self, vals):
         result = super().write(vals)
         if vals.get("auto_orderpoint_template_ids"):
             orderpoint_templates = self.mapped("auto_orderpoint_template_ids")
             orderpoint_templates.create_orderpoints(self)
         return result
 
-    def _compute_historic_quantities_dict(
-        self, location_id=False, from_date=False, to_date=False
-    ):
-        """Returns a dict of products with a dict of historic moves as for
-           a list of historic stock values resulting from those moves. If
-           a location_id is passed, we can restrict it to such location"""
-        location = location_id and location_id.id
-        domain_quant_loc, domain_move_in_loc, domain_move_out_loc = self.with_context(
-            location=location
-        )._get_domain_locations()
-        if not to_date:
-            to_date = fields.Datetime.now()
-        domain_move_in = domain_move_out = [
-            ("product_id", "in", self.ids),
-            ("state", "=", "done"),
-        ] + domain_move_in_loc
-        domain_move_out = [
-            ("product_id", "in", self.ids),
-            ("state", "=", "done"),
-        ] + domain_move_out_loc
+    def _get_stock_move_domain(self, domain_move=False, from_date=False, to_date=False):
```

Comment:
IMHO it would be clearer to use `None` as default here
```suggestion
    def _get_stock_move_domain(self, domain_move=None, from_date=None, to_date=None):
```